configure.ac

```diff
@@ -258,6 +259,11 @@ AS_IF([test "x$enable_asm" != xno], [
             BITTYPE=64
             ASFLAGS="$ASFLAGS -DARCH_X86_64=1"
         ],
+        [aarch64-*|aarch64_be-*|arm64-*|armv8-*], [
```

Comment:
What platforms use a `arm64-` or `armv8-*` host triplet and how do they denote big endian?

Reply:
Removed for now.